includes/polling/os/junos.inc.php

```diff
@@ -21,6 +21,10 @@
 }
 
 $version = snmp_get($device, 'jnxVirtualChassisMemberSWVersion.0', '-Oqv', 'JUNIPER-VIRTUALCHASSIS-MIB');
+if (empty($version)) {
+    preg_match('/kernel JUNOS (\S+),/', $device['sysDescr'], $jun_ver);
+    $version = $jun_ver[1];
```

Comment:
Doesn't setting this here now mean the next check never matches so could be removed.

Reply:
Probably, but I'm not 100% sure all sysDescr are formatted this way and will be matched by the regex. I thought leaving the old code behind as a fallback isn't much of an issue.